<!doctype html><html lang=en-us><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><meta http-equiv=x-ua-compatible content="IE=edge"><meta name=generator content="Source Themes Academic 4.5.0"><meta name=author content="Nicholas Brazeau"><meta name=description content="Projects"><link rel=alternate hreflang=en-us href=https://nicholasbrazeau.com/research/><meta name=theme-color content="#2962ff"><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.6/css/academicons.min.css integrity="sha256-uFVgMKfistnJAfoCUQigIl+JfUaP47GrRKjf6CTPVmw=" crossorigin=anonymous><link rel=stylesheet href=https://use.fontawesome.com/releases/v5.6.0/css/all.css integrity=sha384-aOkxzJ5uQz7WBObEZcHvV5JvRW3TUc2rNPA7pe3AwnsUohiw1Vj2Rgx2KSOkF5+h crossorigin=anonymous><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.css integrity="sha256-ygkqlh3CYSUri3LhQxzdcm0n1EQvH2Y+U5S2idbLtxs=" crossorigin=anonymous><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/styles/darcula.min.css crossorigin=anonymous title=hl-light><link rel=stylesheet href=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/styles/darcula.min.css crossorigin=anonymous title=hl-dark disabled><link rel=stylesheet href="https://fonts.googleapis.com/css?family=Montserrat:400,700%7CRoboto:400,400italic,700%7CRoboto+Mono&display=swap"><link rel=stylesheet href=/css/academic.min.0129a70d91b556bfeee60a2761ebcab2.css><link rel=stylesheet href=/css/academic.96c9d57f78edaefbed63fd2534fb6c17.css><link rel=manifest href=/index.webmanifest><link rel=icon type=image/png href=/img/icon-32.png><link rel=apple-touch-icon type=image/png href=/img/icon-192.png><link rel=canonical href=https://nicholasbrazeau.com/research/><meta property="twitter:card" content="summary_large_image"><meta property="twitter:site" content="@NFBrazeau"><meta property="twitter:creator" content="@NFBrazeau"><meta property="og:site_name" content="Nick Brazeau"><meta property="og:url" content="https://nicholasbrazeau.com/research/"><meta property="og:title" content="Projects | Nick Brazeau"><meta property="og:description" content="Projects"><meta property="og:image" content="https://nicholasbrazeau.com/img/headers/tree.jpg"><meta property="twitter:image" content="https://nicholasbrazeau.com/img/headers/tree.jpg"><meta property="og:locale" content="en-us"><title>Projects | Nick Brazeau</title></head><body id=top data-spy=scroll data-offset=70 data-target=#navbar-main><aside class=search-results id=search><div class=container><section class=search-header><div class="row no-gutters justify-content-between mb-3"><div class=col-6><h1>Search</h1></div><div class="col-6 col-search-close"><a class=js-search href=#><i class="fas fa-times-circle text-muted" aria-hidden=true></i></a></div></div><div id=search-box><input name=q id=search-query placeholder=Search... autocapitalize=off autocomplete=off autocorrect=off spellcheck=false type=search></div></section><section class=section-search-results><div id=search-hits></div></section></div></aside><nav class="navbar navbar-light fixed-top navbar-expand-lg py-0 compensate-for-scrollbar" id=navbar-main><div class=container><a class=navbar-brand href=/>Nick Brazeau</a>
<button type=button class=navbar-toggler data-toggle=collapse data-target=#navbar aria-controls=navbar aria-expanded=false aria-label="Toggle navigation">
<span><i class="fas fa-bars"></i></span></button><div class="collapse navbar-collapse" id=navbar><ul class="navbar-nav ml-auto"><li class=nav-item><a class=nav-link href=/about><span>About</span></a></li><li class=nav-item><a class=nav-link href=/post><span>Blog</span></a></li><li class=nav-item><a class="nav-link active" href=/research data-target=[]><span>Research</span></a></li><li class=nav-item><a class=nav-link href=/publications><span>Pubs & Talks</span></a></li><li class=nav-item><a class=nav-link href=/files/cv.pdf><span>CV</span></a></li><li class=nav-item><a class=nav-link href=/contact><span>Contact</span></a></li><li class=nav-item><a class="nav-link js-search" href=#><i class="fas fa-search" aria-hidden=true></i></a></li></ul></div></div></nav><span class="js-widget-page d-none"></span><section id=header class="home-section wg-blank aboutme-header" style=background-image:linear-gradient(rgba(0,0,0,.2),rgba(0,0,0,.2)),url(/img/headers/chimp_distance.jpeg)><div class=container><div class=row><div class="col-12 col-lg-4 section-heading"></div><div class="col-12 col-lg-8"></div></div></div></section><section id=research class="home-section wg-blank" style=research><div class=container><div class=row><div class="col-12 col-lg-4 section-heading"></div><div class="col-12 col-lg-8"><h1 style=text-align:center>Research & Interests</h1><p><em>Below, I briefly summarize my research to date and elaborate on my interests and what I hope to continue to work on in the future. At the end of this page, you will find current and past projects.</em></p><p> 
 
 
 </p><img align=left width=45% height=200px padding=100 style=padding-top:15%;padding-bottom:15%;padding-right:5% src=/img/idmodel.png><h3 id=infectious-disease-modeling>Infectious Disease Modeling</h3><p>There are many approaches to modeling infectious diseases depending on the research question. In one framework, I have considered the &ldquo;who&rdquo;, &ldquo;where&rdquo;, &ldquo;origin&rdquo; of a pathogen in a region that was previously thought to be resilient to infection. To determine &ldquo;who&rdquo; was being infected, I estimated &ldquo;risk factors&rdquo; accounting for confounding using inverse probability treatment weights estimated through the supervised SuperLearner (machine learning) algorithm; &ldquo;where&rdquo; infections were occurring using Bayesian Gaussian spatial process models; and the &ldquo;origin&rdquo; using basic genetic analyses. For other research questions estimating the severity of an infectious disease, I have employed Bayesian statistical modeling or helped to implement mathematical models for infection scenario and burden planning. As an infectious disease epidemiologist, I am most interested in the intersection of methods, combining molecular techniques, bioinformatics/genomics, evolutionary biology, statistical modeling, and spatial methods to identify pathways of infection and antimicrobial resistance evolution. Using an interdisciplinary approach, I hope to identify interventions that block these pathways.</p><hr><img align=right width=50% height=300px style=padding-top:20%;padding-bottom:15%;padding-left:5% src=/img/phylo.png>
&nbsp;<h3 id=bioinformatics-genomics-statistical-genetics>Bioinformatics, Genomics, Statistical Genetics</h3><p>Molecular surveillance and genomics greatly enrich infectious disease epidemiology analyses, models, and intervention planning. To date, a large majority of my bioinformatic and genomic work has focused on antimicrobial resistance, with an emphasis on putative antimalarial resistance. In the past, I have used genome-wide barcodes to infer how antimalarial selectional pressures have shaped Falciparum malaria population structures in the Democratic Republic of the Congo as well as whole genome sequencing to identify novel antimicrobial mutations in <em>Staphylococcus epidermidis</em>. Separately, I have been engaged in statistical genetics method development with models focused on estimating local inbreeding (<code>DISCent</code>), important for sink-source dynamics and interventional planning, as well as inferring identity by descent in polyclonal malaria infections (<code>polyIBD</code>).<br></p><hr><img align=left width=40% height=200px padding=100 style=padding-top:5%;padding-bottom:5%;padding-right:5% src=/img/world.png>
&nbsp;<h3 id=global-health--medicine-translational-research>Global Health & Medicine, Translational Research</h3><p>Global medicine has become local medicine and vice versa. I am interested in supporting capacity building efforts through research in low- and middle-income countries and historically underserved regions.</p><br><hr></div></div></div></section><section id=projects class="home-section wg-portfolio"><div class=container><div class=margin-auto><div class=center-text><h1 class=mt-0>Projects</h1></div><div><span class="d-none default-project-filter">*</span><div class=project-toolbar><div class=project-filters><div class=btn-toolbar><div class="btn-group flex-wrap"><a href=# data-filter=* class="btn btn-primary btn-lg active">All</a>
<a href=# data-filter=.js-id-active class="btn btn-primary btn-lg">Active</a>
<a href=# data-filter=.js-id-rpackage class="btn btn-primary btn-lg">R Packages</a>
<a href=# data-filter=.js-id-malaria class="btn btn-primary btn-lg">Malaria</a>
<a href=# data-filter=.js-id-covid class="btn btn-primary btn-lg">COVID-19</a>
<a href=# data-filter=.js-id-AMR class="btn btn-primary btn-lg">Antimicrobial Resistance</a>
<a href=# data-filter=.js-id-other class="btn btn-primary btn-lg">Other Projects</a></div></div></div></div><div class="isotope projects-container js-layout-masonry"><div class="project-card project-item isotope-item js-id-AMR js-id-other"><div class=card><a href=/project/sepidapto/ class="card-image hover-overlay"><img src=/project/sepidapto/featured_hu124d2d5a455587247000982b668b9680_72784_550x0_resize_q90_lanczos_3.png alt class=img-responsive></a><div class=card-text><h4><a href=/project/sepidapto/>AMR in the Hospital</a></h4><div class=article-style><p>Case Study of the evolution of Daptomycin in SEpi following prolonged antibiotic exposure</p></div></div></div></div><div class="project-card project-item isotope-item js-id-malaria js-id-AMR"><div class=card><a href=/project/ephiamr/ class="card-image hover-overlay"><img src=/project/ephiamr/featured_hu108ac06c06dac8332d305f59d8f198c3_953996_550x0_resize_q90_lanczos_3.png alt class=img-responsive></a><div class=card-text><h4><a href=/project/ephiamr/>Antimalarial Resistance in Ethiopia</a></h4><div class=article-style><p>Estimating the prevalence of antimalarial mutations in Ethiopia across time and space</p></div></div></div></div><div class="project-card project-item isotope-item js-id-covid"><div class=card><a href=/project/reestimateifr/ class="card-image hover-overlay"><img src=/project/reestimateifr/featured_hu469ca161c71288e1fdc548c14e09dbe9_985711_550x0_resize_q90_lanczos_3.png alt class=img-responsive></a><div class=card-text><h4><a href=/project/reestimateifr/>IFR Re-estimation</a></h4><div class=article-style><p>Estimating the COVID-19 first wave IFR using serologic data accounting for seroreversion</p></div></div></div></div><div class="project-card project-item isotope-item js-id-rpackage js-id-malaria js-id-active"><div class=card><a href=/project/discent/ class="card-image hover-overlay"><img src=/project/discent/featured_hu99837b902f862bcee3ccc4155f9be3da_1553861_550x0_resize_q90_lanczos_3.png alt class=img-responsive></a><div class=card-text><h4><a href=/project/discent/><code>DISCent</code>: Spatial Inbreeding Estimation</a></h4><div class=article-style><p>An algorithm for detecting deme inbreeding spatial coefficients from recombining pathogen genetic data</p></div></div></div></div><div class="project-card project-item isotope-item js-id-malaria"><div class=card><a href=/project/drcpv/ class="card-image hover-overlay"><img src=/project/drcpv/featured_hu6ad0285150bf9aa1bfafcf9f03568686_548051_550x0_resize_q90_lanczos_3.png alt class=img-responsive></a><div class=card-text><h4><a href=/project/drcpv/>Vivax in the DRC</a></h4><div class=article-style><p>Determing who was being infected, where infections were occuring, and where they originated</p></div></div></div></div><div class="project-card project-item isotope-item js-id-rpackage js-id-covid"><div class=card><a href=/project/covidcurve/ class="card-image hover-overlay"><img src=/project/covidcurve/featured_hu453cfaaba1edee59529a4d1056e5895d_57375_550x0_resize_q90_lanczos_3.png alt class=img-responsive></a><div class=card-text><h4><a href=/project/covidcurve/><code>COVIDCurve</code>: COVID-19 Statistical Modeling</a></h4><div class=article-style><p>Bayesian statistical model for estimating age-based infection fatality ratios from serologic data</p></div></div></div></div><div class="project-card project-item isotope-item js-id-rpackage js-id-malaria js-id-active"><div class=card><a href=/project/polysimibd/ class="card-image hover-overlay"><img src=/project/polysimibd/featured_hucd4d833d9dd987e5e5a64a2c279f0389_299030_550x0_resize_q90_lanczos_3.png alt class=img-responsive></a><div class=card-text><h4><a href=/project/polysimibd/><code>polySimIBD</code>: A Simple Malaria Genetic Model</a></h4><div class=article-style><p>Spatial Structured Wright Fisher Model for Malaria Population Genetic Simulations</p></div></div></div></div><div class="project-card project-item isotope-item js-id-malaria js-id-AMR"><div class=card><a href=/project/veritydrc/ class="card-image hover-overlay"><img src=/project/veritydrc/featured_hu6c875dd967a6e13dc7f4d6273dc9e852_213908_550x0_resize_q90_lanczos_3.png alt class=img-responsive></a><div class=card-text><h4><a href=/project/veritydrc/>Antimalarial Resistance in the DRC</a></h4><div class=article-style><p>Exploring population structure attributed to antimalarial resistance in the Democratic Republic of the Congo</p></div></div></div></div><div class="project-card project-item isotope-item js-id-malaria"><div class=card><a href=/project/tzzb_ibd/ class="card-image hover-overlay"><img src=/project/tzzb_ibd/featured_hu56217ff7bc702c6d8254c8d3f71f2ef7_54753_550x0_resize_q90_lanczos_3.png alt class=img-responsive></a><div class=card-text><h4><a href=/project/tzzb_ibd/>Tanzania-Zanzibar Transmission</a></h4><div class=article-style><p>Coupling epidemiologic analysis with relatedness metrics (identity by descent) and population dynamics, we show that there is active …</p></div></div></div></div></div></div></div></div></section><script src=/js/mathjax-config.js></script>
<script src=https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin=anonymous></script>
<script src=https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.4/imagesloaded.pkgd.min.js integrity="sha256-lqvxZrPLtfffUl2G/e7szqSvPBILGbwmsGE1MKlOi0Q=" crossorigin=anonymous></script>
<script src=https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.6/isotope.pkgd.min.js integrity="sha256-CBrpuqrMhXwcLLUd5tvQ4euBHCdh7wGlDfNz8vbu/iI=" crossorigin=anonymous></script>
<script src=https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.js integrity="sha256-X5PoE3KU5l+JcX+w09p/wHl9AzK333C4hJ2I9S5mD4M=" crossorigin=anonymous></script>
<script src=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/highlight.min.js integrity="sha256-aYTdUrn6Ow1DDgh5JTc3aDGnnju48y/1c8s1dgkYPQ8=" crossorigin=anonymous></script>
<script src=https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/languages/r.min.js></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.4/MathJax.js?config=TeX-AMS_CHTML-full" integrity="sha256-GhM+5JHb6QUzOQPXSJLEWP7R73CbkisjzK5Eyij4U9w=" crossorigin=anonymous async></script>
<script>hljs.initHighlightingOnLoad()</script><script>const search_config={indexURI:"/index.json",minLength:1,threshold:.3},i18n={no_results:"No results found",placeholder:"Search...",results:"results found"},content_type={post:"Posts",project:"Projects",publication:"Publications",talk:"Talks"}</script><script id=search-hit-fuse-template type=text/x-template>
      <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
      </div>
    </script><script src=https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.2.1/fuse.min.js integrity="sha256-VzgmKYmhsGNNN4Ph1kMW+BjoYJM2jV5i4IlFoeZA9XI=" crossorigin=anonymous></script>
<script src=https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin=anonymous></script>
<script src=/js/academic.min.01f68d44d80310e669a1731b68a35481.js></script><div class=container><footer class=site-footer><p class=powered-by>© 2022 Nicholas Brazeau &#183;
<a href=https://github.com/nickbrazeau/nickbrazeau.github.io target=_blank rel=noopener><i class="fas fa-code-branch" style=color:#0072b2></i></a> &#183;
Created by a large hominin in between <i class="fas fa-stethoscope" style=color:#0072b2></i> and <i class="fas fa-laptop" style=color:#0072b2></i>
<span class=float-right aria-hidden=true><a href=# id=back_to_top><span class=button_icon><i class="fas fa-chevron-up fa-2x"></i></span></a></span></p></footer></div><div id=modal class="modal fade" role=dialog><div class=modal-dialog><div class=modal-content><div class=modal-header><h5 class=modal-title>Cite</h5><button type=button class=close data-dismiss=modal aria-label=Close>
<span aria-hidden=true>&#215;</span></button></div><div class=modal-body><pre><code class="tex hljs"></code></pre></div><div class=modal-footer><a class="btn btn-outline-primary my-1 js-copy-cite" href=# target=_blank><i class="fas fa-copy"></i> Copy</a>
<a class="btn btn-outline-primary my-1 js-download-cite" href=# target=_blank><i class="fas fa-download"></i> Download</a><div id=modal-error></div></div></div></div></div></body></html>